test/compress_test.js

```diff
@@ -23,6 +23,8 @@ exports.compress = {
       actual.push(entry.path);
     });
     parse.on('close', function() {
+      actual.sort();
+      expected.sort();
```

Comment:
I'm confused about this. A zip file is an ordered list of files, it has concept of a first file and a last file. The order is specified when the zip file is created or when new files are added to it.

With that in mind, I don't think it's necessary to sort anything here. Unless the `compress_test_files.zip` file is created by some unknow (random) process where you don't know the ordering of the files?